src/libs/HapticFeedback/types.ts

```diff
@@ -0,0 +1,6 @@
+type HapticFeedbackPress = () => void;
+type HapticFeedbackLongPress = () => void;
+type HapticFeedbackSuccess = () => void;
+type HapticFeedbackError = () => void;
+
+export type {HapticFeedbackPress, HapticFeedbackLongPress, HapticFeedbackSuccess, HapticFeedbackError};
```

Comment:
Same as [this PR](https://github.com/Expensify/App/pull/26771), what about defining a type for the whole file structure?

```ts
type HapticFeedback = {
    press: () => void;
    longPress: () => void;
    success: () => void;
    error: () => void;
};

const hapticFeedback: HapticFeedback = {
    press: () => {},
    longPress: () => {},
    success: () => {},
    error: () => {},
};

export default hapticFeedback;
```

Reply:
Done :) @fabioh8010